dbsync-central/application.prod.properties

```diff
@@ -0,0 +1,182 @@
+
+# Dialect of the openMRS datasource (should not be changed in a usual use)
+spring.openmrs-datasource.dialect=org.hibernate.dialect.MySQLDialect
+
+# Url of the openMRS datasource, you don't have to change this value as long you've set the placeholder property values
+spring.openmrs-datasource.jdbcUrl=jdbc:mysql://${openmrs.db.host}:${openmrs.db.port}/${openmrs.db.name}
+
+# User name of the openMRS datasource
+spring.openmrs-datasource.username=${spring_openmrs_datasource_user}
+
+# Password of the openMRS datasource
+spring.openmrs-datasource.password=${spring_openmrs_datasource_password}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Configuration of the Management database *****************************************************
+#
+# Driver class of the management datasource (should not be changed in a usual use)
+spring.mngt-datasource.driverClassName=com.mysql.cj.jdbc.Driver
+
+# Dialect of the management datasource (should not be changed in a usual use)
+spring.mngt-datasource.dialect=org.hibernate.dialect.MySQLDialect
+
+# Url of the management datasource
+spring.mngt-datasource.jdbcUrl=jdbc:mysql://${openmrs.db.host}:${openmrs.db.port}/${dbsync_name}
+
+# User name of the management datasource
+spring.mngt-datasource.username=${spring_openmrs_datasource_user}
+
+# Password of the management datasource
+spring.mngt-datasource.password=${spring_openmrs_datasource_password}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** ActiveMQ configuration ***********************************************************************
+#
+# Broker URL of the Artemis MQ broker shared with the receiver (SHOULD include the protocol e.g. tcp://192.168.1.2:61616)
+#spring.artemis.brokerUrl=tcp://artemis:61616
+#
+spring.artemis.host=${spring_artemis_host}
+spring.artemis.port=${spring_artemis_port}
+# User name of the Artemis broker
+spring.artemis.user=${spring_artemis_user}
+#
+# Password of the Artemis broker
+spring.artemis.password=${spring_artemis_password}
+# ----------------------------------------------------------------------------------------------------------------------
+
+
+# *********************** Configuration of the ArtemisMQ endpoint to read from inbound DB sync payloads ****************
+# Use either one of the following confs. JMS endpoint if activeMQ is configured or file endpoint for TESTING purposes
+# Camel JMS Endpoint e.g. activemq:topic:<topic-name> or FILE Endpoint e.g. file:<folderPath> Should be absolute path
+camel.input.endpoint=activemq:topic:openmrs.sync.topic?subscriptionDurable=true&durableSubscriptionName=DB-SYNC-RECEIVER&connectionFactory=activeMqConnFactory&acknowledgementModeName=CLIENT_ACKNOWLEDGE&messageListenerContainerFactory=customMessageListenerContainerFactory
```

Comment:
I think for new province(not the current one that we have been working with), we should go back to using a queue since in your broker.xml file you disabled auto deletion of queues which addresses the issue that made us switch to queues

Reply:
And in that case you can drop the **subscriptionDurable** and **durableSubscriptionName** properties

Reply:
This has any impact on senders? We are installing the central server from scratch

Reply:
What am saying is that we are going to use the docker setup for the rest of the future provinces, it would be nice to have this by default setup with a queue and then for our existing province, you will just update the property value to be a topic that way its remote sites don't have to touched

Reply:
Ok